src/robot/model/tagstatistics.py

```diff
@@ -71,6 +71,13 @@ def _add_to_combined_statistics(self, test):
             if stat.match(test.tags):
                 stat.add_test(test)
 
+    def _suppress_reserved(self, tag):
+        # don't suppress reserved tags if the user explicitly included them
+        if tag.startswith('robot:') and not (self._included and self._included.match(tag)):
+            return True
+        else:
+            return False
```

Comment:
This can be simplified to just
```python
return tag.startswith('robot:') and not self._included.match(tag)
```
Not 100% sure can that `self._included and` part be removed but I think it can.

Reply:
I am not sure either.. I took this pattern from 
```
    def _is_included(self, tag):
        if self._included and not self._included.match(tag):
            return False
        return not self._excluded.match(tag)
```
and when I change the above to 
```
        if not self._included.match(tag):
            return False
```
some acceptance tests fail (have not checked why)

I can simplify my code as suggested by you (and a _doh!_ for the stupid if), the tests will pass..  I'll change as suggested..